gluten-core/src/main/scala/io/glutenproject/expression/NaryExpressionTransformer.scala

```diff
@@ -19,11 +19,11 @@ package io.glutenproject.expression
 import io.glutenproject.expression.ConverterUtils.FunctionConfig
 import io.glutenproject.substrait.expression.{ExpressionBuilder, ExpressionNode}
 
-import org.apache.spark.sql.catalyst.expressions.Expression
+import org.apache.spark.sql.catalyst.expressions._
 
 import com.google.common.collect.Lists
 
-class BasicCollectionOperationTransfomer(
+class NaryExpressionTransformer(
```

Comment:
Thanks for your efforts!
It looks we can just make it a case class. Then, there is no need to define `object NaryExpressionTransformer`. Right?

We may do the same change for many transformer classes, each of which extends `ExpressionTransformer` and is not extended by other class.

Reply:
OK, I will remove `object NaryExpressionTransformer` and check other transformer classes later on.